File: src/main/java/com/elf/vipForElf/domain/realEstateListing/vo/RealEstateListingListDTO.java
package com.elf.vipForElf.domain.realEstateListing.vo;

import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.List;

@Getter
@NoArgsConstructor
public class RealEstateListingListDTO {
    Long totalCount;
    List<RealEstateListingRow> realEstateListingRowList;

    public RealEstateListingListDTO(Long totalCount, List<RealEstateListingRow> realEstateListingRowList) {
        this.realEstateListingRowList = realEstateListingRowList;
        this.totalCount = totalCount;
    }
}

File: src/main/java/com/elf/vipForElf/web/dto/RealEstateListingListDTO.kt
package com.elf.vipForElf.web.dto

import com.elf.vipForElf.domain.realEstateListing.vo.RealEstateListingRow

data class RealEstateListingListDTO(
    var totalCount: Long? = null,
    var realEstateListingRowList: List<RealEstateListingRow>? = null
)
